test/controllers/api/v1/habits_controller_test.exs

```diff
@@ -11,21 +11,42 @@ defmodule Habits.API.V1.HabitControllerTest do
       today_check_in = Factory.insert(:check_in, habit: habit, date: today_date)
       Factory.insert(:check_in, habit: habit, date: yesterday_date)
 
+      # Should we actually use the TokenAuthentication plug here?
```

Comment:
Nope! Upon reflection, I like how authentication is encapsulated by a plug and the controller doesn't need to care about it. This means I could swap out a different kind of authentication and most of my controller tests wouldn't need to change.